front/app/containers/Admin/projects/project/analysis/Tags/AutoTaggingModal/Step1.tsx

```diff
@@ -0,0 +1,271 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { TagType } from 'api/analysis_tags/types';
+
+import {
+  Box,
+  Title,
+  Text,
+  colors,
+  Spinner,
+  IconTooltip,
+  Icon,
+  Label,
+  Radio,
+} from '@citizenlab/cl2-component-library';
+import Tag from '../Tag';
+import { AutoTaggingMethod } from 'api/analysis_background_tasks/types';
+import FilterItems from '../../FilterItems';
+import { IInputsFilterParams } from 'api/analysis_inputs/types';
+import { isEmpty } from 'lodash-es';
+
+const AutoTagMethodContainer = styled.div<{ disabled: boolean }>`
+  background-color: ${colors.grey100};
+  border-radius: 3px;
+  padding: 16px;
+  ${({ disabled }) =>
+    disabled
+      ? ''
+      : `
+      cursor: pointer;
+      &:hover {
+        border: 1px black;
+        box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px,
+          rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+      }`}
+`;
+
+const AutoTagTargetContainer = styled.div`
+  flex: 1;
+  background-color: ${colors.grey100};
+  border-radius: 3px;
+  padding: 16px;
+  cursor: pointer;
+  &:hover {
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px,
+      rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+  }
+  &.selected {
+    outline: 1px solid ${colors.grey400};
+  }
+  &.disabled {
+    cursor: not-allowed;
+    opacity: 0.5;
+    box-shadow: none;
+  }
+`;
+const AutoTagOption = ({
+  children,
+  tagType,
+  title,
+  onSelect,
+  disabled,
+  isLoading,
+  tooltip,
+}: {
+  children: ReactNode;
+  tagType: TagType;
+  title: string;
+  onSelect: () => void;
+  disabled: boolean;
+  isLoading: boolean;
+  tooltip?: string;
+}) => {
+  return (
+    <AutoTagMethodContainer onClick={() => onSelect()} disabled={disabled}>
```

Comment:
Divs with onClick should behave as buttons - this normally means adding `tabIndex={0}` to the component and making sure the enter key performs the same action as a click when the element is focused. Something like:
```            
onKeyDown={(e) => {
    if (e.key === 'Enter') {
        onSelect();
    }
}}
```

Reply:
Clear, will take it up in the bug fixing round 2